app/views/draftmetadata/draftMetadataUpload.scala.html

```diff
@@ -5,12 +5,37 @@
 @import java.util.UUID
 @(consignmentId: UUID, consignmentRef: String, frontEndInfo: FrontEndInfo, name: String)(implicit request: RequestHeader, messages: Messages)
 
-@main("[WIP] Draft Metadata Upload", name = name) {
-    <div id="draft-metadata-upload" class="govuk-grid-row">
+@main("[WIP] Draft Metadata Upload", name = name, backLink = Some(backLink(routes.AdditionalMetadataEntryMethodController.additionalMetadataEntryMethodPage(consignmentId).url, "How would you like to enter metadata?"))) {
```

Comment:
The main template will need updating with the correct header. `[WIP] Draft Metadata Upload` was a placeholder for the skeleton page